app-admin/logcheck/logcheck-1.4.3-r5.ebuild

```diff
@@ -67,7 +68,7 @@ src_install() {
 
 	if use cron; then
 		exeinto /etc/cron.hourly
-		newexe "${FILESDIR}"/"${PN}"-r4.cron ${PN}
+		newexe "${FILESDIR}"/"${PN}"-r5.cron ${PN}
```

Comment:
Why? You don't need to keep the version of the cron aligned with the ebuild.

Reply:
No special reason, It is just to be sure I will not to remove a needed file when -r4 will be dropped.